app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+
+    can %i[index show], User
+    can %i[new create show index], Post
+    can %i[new create show index], Comment
+    can %i[update destroy], Post, { author_id: user.id }
+    can %i[update destroy], Comment, { author_id: user.id }
+    can :manage, Post if user.is? :admin
+    can :manage, Comment if user.is? :admin
```

Comment:
- [ ] Inspecting your ability class, I can see that you correctly have the rule that specifies that a user can manage the update or deletes for a comment if they are the owner of the user. I therefore believe that the previous reviewer @tresorsawasawa  made a mistake when requiring this change.

**I tested the project locally testing the comments as an example:** I have two users, "Nemwel" and "Mimi". Nemwel has a post and has a comment. Mimi also created a comment in Nemwel's post. When I sign in as Nemwel, I can only delete my own comment and not the one created by Mimi. Pictured below:

![image](https://github.com/andrianarivo/blog_app/assets/86318284/90cde106-4ef1-4b68-a97a-849a0253623f)


Equally, what the reviewer mentioned "user can delete a post if it is theirs or if they have an admin role and can delete a comment if it is theirs or if they have an admin role," all these rules have been implemented in the ability class. @tresorsawasawa  kindly recheck the ability class and you will find the ability rules have been included there.

Reply:
Sorry for being impatient, I'm doing my best to contain myself. I guess it's part of the learning process. Thank you

Reply:
@Nemwel-Boniface I am still not able to submit my project, it's been 12 hours now

Reply:
@andrianarivo It was approved. KIndly refresh your dashboard and let me know if you are still not able to submit.

Reply:
@Nemwel-Boniface good morning. from my dashboard  I cannot submit. We are peer programming.